sdk/servicebus/Azure.Messaging.ServiceBus/tests/Administration/ServiceBusManagementClientLiveTests.cs

```diff
@@ -67,10 +74,13 @@ private ServiceBusAdministrationClient CreateSasTokenClient()
         }
 
         [Test]
-        public async Task BasicQueueCrudOperations()
+        [TestCase(false)]
```

Comment:
Any chance that we can make the `premium` part optional if the environment variable isn't set?   While I agree its important to test this with the nightly runs,  I don't see a ton of value in doing so for the local dev loop by default.    

With the premium namespace being expensive and test resources being cleaned up on a delay, it seems like we'd be potentially wasting a bunch of money.   Likewise for community contributors who likely don't have cleanup running and may not be aware or remember to manually delete.

Reply:
Good point - when you say env var, do you mean the CI env var?

Reply:
or do you mean the new env var?

Reply:
I was meaning the  `SERVICEBUS_PREMIUM_NAMESPACE_CONNECTION_STRING` value being unset or empty.    That would let us run locally on an opt-in basis, but not fail the test with an error if we didn't opt-in.

Reply:
Well it would work sometimes - if you already have other env vars, but not this one then yes. But if you don't have any of the env vars, it would still run through the resource creation and create the premium namespace.

Reply:
Fair point, but deploying Test Resources will create any regardless....   unless we update the test resources ARM to conditionally create it?

Reply:
I think it is worth looking into for MQ across languages.